docs/services.mdx

```diff
@@ -23,6 +23,7 @@ route: /services
 
 - [uPort](https://www.uport.me/)
 - [Ion](https://github.com/decentralized-identity/ion/)
+- [W3C DIDs spec](https://w3c-ccg.github.io/did-spec/)
```

Comment:
What do you think about `[A-z]` sorting these links since you are already in there? 🤔

 Looks great otherwise!

Reply:
ooh yea i will do that, thanks for the suggestion!